flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/RowCsvInputFormatSplitTest.java

```diff
@@ -0,0 +1,119 @@
+package org.apache.flink.formats.csv;
+
+import org.apache.flink.api.common.typeinfo.BasicTypeInfo;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.java.typeutils.RowTypeInfo;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.core.fs.FileInputSplit;
+import org.apache.flink.types.Row;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.apache.flink.formats.csv.RowCsvInputFormatTest.PATH;
+import static org.apache.flink.formats.csv.RowCsvInputFormatTest.createTempFile;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test split logic for {@link RowCsvInputFormat}.
+ */
+public class RowCsvInputFormatSplitTest {
```

Comment:
I'm wondering if these tests are sufficient. What about strings sourrounded by `"` or escaped delimiters? Could you copy more tests around escaping for the deserialization schema tests?